driver-mct/cime_config/config_component_e3sm.xml

```diff
@@ -402,6 +401,8 @@
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%FRISwISC01to60E3r1">1440</value>
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%EC30to60E2r2">48</value>
       <value compset="_DATM.*_SLND.*MPAS" grid="oi%WC14to60E2r3">48</value>
+      <value compset="_DATM.*_SLND.*MPAS" grid="oi%IcoswISC30E3r5">48</value>
+      <value compset="_DATM.*_SLND.*MPAS" grid="oi%SOwISC12to30E3r4">48</value>
```

Comment:
Let's also add `SOwISC12to30E3r3`.

And let's add similar entries for `OCN_NCPL`.

Reply:
My understanding is that these apply to all G or GG cases, which is why I think `SOwISC12to30E3r3` should also be included.  Does that seem right?

Reply:
I did the OCN_NCPL change already. Are we still working with the SOwISC12to30E3r3? I wasn't sure if it had been replaced by the SOwISC12to30E3r4

Reply:
No, the `SOwISC12to30E3r4` is strictly for the FAnSSIE project.  Polar is still working only with `SOwISC12to30E3r3`.  Both are actively being used.

Reply:
For context, `SOwISC12to30E3r4` uses topography from MALI that is appropriate for eventual glc-ocn coupling, `SOwISC12to30E3r3` uses observation-derived topography that has been more thoroughly vetted.

Reply:
OK, I'll add it as well

Reply:
Thanks!